content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+example:
+
+- [X.509 certificate](https://en.wikipedia.org/wiki/X.509)
+- [JWT](https://en.wikipedia.org/wiki/JSON_Web_Token)
+
+Each identity Istio assigns is understood throughout the mesh and you can use it
+to enable mutual authentication and enforce policies.
+
+An identity contains a substring identifying the mesh that created it. For
+example in `spiffe://mytrustdomain.com/ns/default/sa/myname` the substring
+identifying the mesh is: `mytrustdomain.com`. We call this substring the {{<
+gloss>}}trust domain{{< /gloss >}} of the mesh. Every Istio mesh has a globally
+unique trust domain used to create identity names.
+
+The Certificate Authority (CA) creates the identities and their key material.
+You can verify the message sender's identity with the public key of the CA that
+created the identity. A **trust bundle** is the set of
+all CA public keys used by an Istio mesh. With a mesh's trust bundle, anyone can
+verify the sender of any message coming from that mesh.
+
+### Trust within a mesh
+
+Within a single Istio mesh, a CA creates identities and ensures that each
+workload instance has the key material needed to recognize all identities in the
+mesh. This model allows workload instances in the mesh to authenticate each
+other when communicating.
+
+{{< image width="50%"
+    link="./single-trust.svg"
+    alt="A service mesh with a certificate authority"
+    title="Trust within a mesh"
+    caption="A service mesh with a certificate authority"
+    >}}
+
+### Trust between meshes
+
+If a service in a mesh requires a service in another, you must federate identity
+and trust between the two meshes. To federate identity and trust,, you must
+exchange the trust bundles of the meshes. You can exchange the trust bundles
+either manually or automatically using a protocol such as [SPIFFE Trust Domain Federation](https://docs.google.com/document/d/1OC9nI2W04oghhbEDJpKdIUIw-G23YzWeHZxwGLIkB8k/edit).
+Once you import a trust bundle to a mesh, you can configure local policies for
+those identities.
+
+{{< image width="50%"
+    link="./multi-cluster.svg"
+    alt="Multiple service meshes with certificate authorities"
+    title="Trust between meshes"
+    caption="Multiple service meshes with certificate authorities"
+    >}}
+
+## Tenancy models
+
+In Istio, a **tenant** is a group of users that share
+common access and privileges to a set of deployed workloads. Generally, you
+isolate the workload instances from multiple tenants from each other through
+network configuration and policies.
+
+You can configure tenancy models to satisfy the following organizational
+requirements for isolation:
+
+- Security
+- Policy
+- Capacity
+- Cost
+- Performance
+
+Istio supports two types of tenancy models:
+
+- [Namespace tenancy](#namespace-tenancy)
+- [Cluster tenancy](#cluster-tenancy)
+
+### Namespace tenancy
+
+In Istio, a [namespace](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-namespace)
```

Comment:
Is the term `namespace`  and `namespace tenancy` supported in non-Kubernetes environments?

Reply:
They are for mesh expansion, although that is kind of cheating since it is based on a k8s foundation. But that's a really good point that I have no idea how to resolve, since a lot of how meshes work is (overly) tied to k8s. Maybe we can somehow mention that Istio also works in non-k8s environments but we don't have the tenancy models defined well there yet...